content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -4,4 +4,39 @@ description: Istio 1.4 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- **Added** support for fractional [request mirroring](/docs/tasks/traffic-management/mirroring/).
+- **Improved** the Envoy sidecar to exit upon crashing, giving better visibility of health.
+- **Improved** Pilot to skip sending redundant configuration to Envoy when no changes are required.
+
+## Security
+
```

Comment:
Can we add:
```
- **Added** v1beta1 authorization policy for access control which deprecates the v1alpha1 RBAC policy.
```

Reply:
@yangminzhu definitely needs to be added at some point, I was waiting for https://github.com/istio/istio.io/issues/5289 before adding this. Unlike previous release notes PRs I am trying to split this up and just get the notes that are ready in asap and we can continue to add others as they are ready

Reply:
Sounds good, we can update this later when other authz-v2 docs are merged.